### ISSUE TITLE
General discussion

### ISSUE THREAD

**Issue by seblindfors** [topher-au/WoWmapper#1]
General discussion thread
---------------------------------------------------------------
- This thread exists to provide feedback and new ideas.
- Don't post about an issue in this thread.
- Check closed issues before you open a new issue.

**Comment by seblindfors** [comment]
After an hour of play I have just one pointer that might be worth addressing.
Given a new ability on level up, I couldn't hold down RS to drag the ability away from the bar.
If you're deliberately firing the up state immediately after the down state, I would change that and make it fire the up state on release instead. This is only necessary for the downpress on the sticks though.

Another thing to note is the lack of battery information. When running on bluetooth, it could be nice to get some kind of indication of low power. The official DS4Windows profile changes the lightbar to a flashing red when the battery is low. Maybe a similar solution could be applied here?

Other than that, I think what you've got so far is pretty solid. I even tried the advanced haptics to see how it worked and I kinda liked the "vibrate on damage taken" option.

**Comment by topher-au** [comment]
> Given a new ability on level up, I couldn't hold down RS to drag the
ability away from the bar.

Easy fix, will implement in the next version. Didn't consider that scenario
when I was coding it :P I have spent most of my time alt-tabbed in
Orgrimmar rather than playing :P

> Another thing to note is the lack of battery information. When running on
bluetooth, it could be nice to get some kind of indication of low power.
The official DS4Windows profile changes the lightbar to a flashing red when
the battery is low.

Can definitely work on that. I have been using USB so I hadn't thought much
about it but the functionality is all there, just gotta put it into a
pretty package :)

> Other than that, I think what you've got so far is pretty solid. I even
tried the advanced haptics to see how it worked and I kinda liked the
"vibrate on damage taken" option.

Thanks :) I thought the rumble was a nice touch, and since it's fairly
trivial to pull that info from memory if you know where to look, it
shouldn't be too difficult to keep on top of when a new patch is released.

On Mon, Nov 30, 2015 at 1:56 PM, Sebastian Lindfors <
notifications@github.com> wrote:

> After an hour of play I have just one pointer that might be worth
> addressing.
> Given a new ability on level up, I couldn't hold down RS to drag the
> ability away from the bar.
> If you're deliberately firing the up state immediately after the down
> state, I would change that and make it fire the up state on release
> instead. This is only necessary for the downpress on the sticks though.
>
> Another thing to note is the lack of battery information. When running on
> bluetooth, it could be nice to get some kind of indication of low power.
> The official DS4Windows profile changes the lightbar to a flashing red when
> the battery is low. Maybe a similar solution could be applied here?
>
> Other than that, I think what you've got so far is pretty solid. I even
> tried the advanced haptics to see how it worked and I kinda liked the
> "vibrate on damage taken" option.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/topher-au/DS4ConsolePort/issues/1#issuecomment-160504472>
> .
>



-- 
Thankyou,
Chris Sheridan <lordramza@gmail.com>

**Comment by seblindfors** [comment]
Also, this check for bound keys could be improved:
![image](https://cloud.githubusercontent.com/assets/7464938/11469381/36bfb57a-9756-11e5-8b6d-92cb0f789b04.png)

How about instead of showing that warning, just swap the bindings and say they were swapped?
Right now, if I wanted to set Shift to another modifier, I would have to open the old modifier first and set it to something unrelated, then go into the new modifier and set Shift and then back to the old modifier to set that binding to what the new modifier had previously.

**Comment by topher-au** [comment]
Yeah that's an issue, I just wanted to get something in place to start off with. It's on the list :)

**Comment by seblindfors** [comment]
Also, in the spirit of not getting banned, I should mention that this type of brute force inut is not allowed:
![image](https://cloud.githubusercontent.com/assets/7464938/11475140/0f4aaaec-977c-11e5-9d2f-6f2e5b25d06f.png)

I'm fairly certain that all input that is not 1:1 is considered automation, which violates 2A under the terms of use. 

> You agree that you will not, under any circumstances: 
>A. use cheats, automation software (bots), hacks, mods or any other unauthorized third-party >software designed to modify the World of Warcraft experience;

In regards to what Blizzard has to say about the haptic feedback, maybe asking for permission is actually a good idea?
 
>C. use any unauthorized third-party software that intercepts, “mines,” or otherwise collects information from or through the Game or the Service, including without limitation any software that reads areas of RAM used by the Game to store information about a character or the game environment; provided, however, that Blizzard may, at its sole and absolute discretion, allow the use of certain third party user interfaces;

**Comment by seblindfors** [comment]
When the key state thing is fixed, I will go ahead and put this up as the recommended software for DS4 instead of DS4Windows. I'm only using this now. :wink:

**Comment by topher-au** [comment]
Must have done a good job haha. Grab the 0.1.1 source and see if that does
the trick :)

**Comment by topher-au** [comment]
Didn't read the full comment thread, i'll look into the key states thing tomorrow :)

**Comment by seblindfors** [comment]
Yeah, I tend to not think things through before posting so I end up doing a lot of retractive editing. Github doesn't send a new mail when comments are updated though. :smile: 

I posted a video using this tool btw:
https://youtu.be/cTN_-Nl7Gns

**Comment by topher-au** [comment]
Looks good! Should have the key state thing fixed later tonight.

Have you considered making a custom action bar add-on? I'm looking into it
but have no idea where to start coding for WoW, any suggestions?

**Comment by seblindfors** [comment]
I actually know a lot about that since ConsolePort is using hidden action buttons to perform everything that's not in the official bindings. Everytime you click something with the cursor for example, you're using a hidden action button. What kind of action bar addon are you thinking of?

**Comment by topher-au** [comment]
Basically a cross action bar like final fantasy xiv, if you don't know how
it works I can make a video tonight

**Comment by topher-au** [comment]
Add me on battle.net Topher#1992

**Comment by seblindfors** [comment]
I've added you. WIll be working on a new config frame today, though.

**Comment by seblindfors** [comment]
If it's not too much hassle, can you upload another release in zip format with an appropriate name?
Some new user was confused and downloaded the old version because of the whole Debug.rar thing.

**Comment by Wiebbe** [comment]
So, i was wondering if WoWmapper should be able to bind the guide button as well? I'm running Windows 10, closed Steam permanently (even from the processes window) and turned off the Game DVR. But somehow the Guide button does not seem to work. 

Anyway I could check if something was interfering with the button?

**Comment by topher-au** [comment]
Hey there! Sorry didn't see this at the bottom of the general discussions :)

It should be bindable! If you're on W10 w/ Steam exited and the Game bar shortcut disabled it should work. Press Windows+G, then settings, then disable the Guide button hotkey - if you still have issues let me know :)

**Comment by evans95** [comment]
hey having a problem thinking it was net framework but i have updated to latest and run repair tool and still having the same problem this pops up when i open wowmapper and i cant go into plugins as this just pops up how do i fix it?
unhandled exception has occurred in your application
could not load file or assembly input_ds4 version=1.4.32.0
culture=neutral public key token= null or one of its dependencies 
operation is not supported (exception from hresult: 0x80131515 

also ill add the details below 

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'input_ds4, Version=1.4.32.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'input_ds4, Version=1.4.32.0, Culture=neutral, PublicKeyToken=null' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at WoWmapper.MainForm.InitializePlugin(String DllFile)
   at WoWmapper.MainForm.MainForm_Load(Object sender, EventArgs e)
   at System.Windows.Forms.Form.OnLoad(EventArgs e)
   at System.Windows.Forms.Form.OnCreateControl()
   at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   at System.Windows.Forms.Control.CreateControl()
   at System.Windows.Forms.Control.WmShowWindow(Message& m)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at WoWmapper.MainForm.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)




************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
WoWmapper
    Assembly Version: 0.4.3.0
    Win32 Version: 0.4.3.0
    CodeBase: file:///C:/Users/Kyle/Desktop/WOWM/WoWmapper.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Accessibility
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/Accessibility/v4.0_4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///C:/Users/Kyle/Desktop/WOWM/Newtonsoft.Json.DLL
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
WoWmapper_Input
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Users/Kyle/Desktop/WOWM/WoWmapper_Input.DLL
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.

**Comment by evans95** [comment]
hey having a problem thinking it was net framework but i have updated to latest and run repair tool and still having the same problem this pops up when i open wowmapper and i cant go into plugins as this just pops up how do i fix it?
unhandled exception has occurred in your application
could not load file or assembly input_ds4 version=1.4.32.0
culture=neutral public key token= null or one of its dependencies 
operation is not supported (exception from hresult: 0x80131515

also ill add the details below

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'input_ds4, Version=1.4.32.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'input_ds4, Version=1.4.32.0, Culture=neutral, PublicKeyToken=null' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
at WoWmapper.MainForm.InitializePlugin(String DllFile)
at WoWmapper.MainForm.MainForm_Load(Object sender, EventArgs e)
at System.Windows.Forms.Form.OnLoad(EventArgs e)
at System.Windows.Forms.Form.OnCreateControl()
at System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
at System.Windows.Forms.Control.CreateControl()
at System.Windows.Forms.Control.WmShowWindow(Message& m)
at System.Windows.Forms.Control.WndProc(Message& m)
at WoWmapper.MainForm.WndProc(Message& m)
at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)

**Comment by seblindfors** [comment]
You should post this in a separate issue, not in the general discussion thread.

**Comment by evans95** [comment]
oh i see sorry did not see the new issue button

**Comment by topher-au** [comment]
Hey mate, please check the plugins folder, that both files exist.
Additionally, right click each item and go to properties, and if there is a
box that says "unblock", click it to unblock the file. This should solve
the problem

**Comment by donpake** [comment]
hey mate, 

I'm having the same issue as Evans95, I unblocked both files and now Wowmapper just crashes when i run it,I tried running as admin still didn't work also tried disabling my anit virus but to no avail. any help would be much appreciated.

Cheers.

**Comment by evans95** [comment]
Hey thanks for the reply sorry i moved the comment i made it in a new issue but i have went through all of them and they are all unblocked and both files are in the plugin folder and also unblocked but still crashes on me

**Comment by topher-au** [comment]
Do you have SlimDX 4.0 installed?

**Comment by evans95** [comment]
Yeah installed even ran a repair on it and net framework latest edition all dll unblocked and there. tried running as admin followed install instructions to a tee just keeps crashing on me not sure whats causing  it

**Comment by topher-au** [comment]
Can you post a crash log, a screenshot or some more info? The two things I have told you seemed to fix everyone else's issues so far.

**Comment by evans95** [comment]
Log Name:      Application
Source:        Application Error
Date:          06/01/2016 10:09:55
Event ID:      1000
Task Category: (100)
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      Kyle-PC
Description:
Faulting application name: WoWmapper.exe, version: 0.4.3.0, time stamp: 0x568a1002
Faulting module name: KERNELBASE.dll, version: 6.1.7601.19045, time stamp: 0x56259271
Exception code: 0xe0434352
Fault offset: 0x000000000000b3dd
Faulting process id: 0x1404
Faulting application start time: 0x01d1486a3cf3b6b6
Faulting application path: C:\Users\Kyle\AppData\Local\WoWmapper\WoWmapper.exe
Faulting module path: C:\Windows\system32\KERNELBASE.dll
Report Id: a1f62b11-b45d-11e5-aab4-ce3ef6f819b5
Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name="Application Error" />
    <EventID Qualifiers="0">1000</EventID>
    <Level>2</Level>
    <Task>100</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-01-06T10:09:55.000000000Z" />
    <EventRecordID>2702</EventRecordID>
    <Channel>Application</Channel>
    <Computer>Kyle-PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>WoWmapper.exe</Data>
    <Data>0.4.3.0</Data>
    <Data>568a1002</Data>
    <Data>KERNELBASE.dll</Data>
    <Data>6.1.7601.19045</Data>
    <Data>56259271</Data>
    <Data>e0434352</Data>
    <Data>000000000000b3dd</Data>
    <Data>1404</Data>
    <Data>01d1486a3cf3b6b6</Data>
    <Data>C:\Users\Kyle\AppData\Local\WoWmapper\WoWmapper.exe</Data>
    <Data>C:\Windows\system32\KERNELBASE.dll</Data>
    <Data>a1f62b11-b45d-11e5-aab4-ce3ef6f819b5</Data>
  </EventData>
</Event>

And 

Log Name:      Application
Source:        .NET Runtime
Date:          06/01/2016 10:09:54
Event ID:      1026
Task Category: None
Level:         Error
Keywords:      Classic
User:          N/A
Computer:      Kyle-PC
Description:
Application: WoWmapper.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException
   at WoWmapper.MapperModule.InputMonitor.MouseThread()
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart()


Event Xml:
<Event xmlns="http://schemas.microsoft.com/win/2004/08/events/event">
  <System>
    <Provider Name=".NET Runtime" />
    <EventID Qualifiers="0">1026</EventID>
    <Level>2</Level>
    <Task>0</Task>
    <Keywords>0x80000000000000</Keywords>
    <TimeCreated SystemTime="2016-01-06T10:09:54.000000000Z" />
    <EventRecordID>2701</EventRecordID>
    <Channel>Application</Channel>
    <Computer>Kyle-PC</Computer>
    <Security />
  </System>
  <EventData>
    <Data>Application: WoWmapper.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.IO.FileNotFoundException
   at WoWmapper.MapperModule.InputMonitor.MouseThread()
   at System.Threading.ExecutionContext.RunInternal(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object, Boolean)
   at System.Threading.ExecutionContext.Run(System.Threading.ExecutionContext, System.Threading.ContextCallback, System.Object)
   at System.Threading.ThreadHelper.ThreadStart()

</Data>
  </EventData>
</Event>

ill send some photos next i took them on phone for ease as cropping screenshots is a pain over 3 screens the screenshot is huge but they are clear and might give some insight.

i think this is the logs your asking for if not please inform me how to find them i used event viewer these are the 2 errors that show in event viewer when i open wowmapper.

**Comment by evans95** [comment]
![20160106_101828](https://cloud.githubusercontent.com/assets/16563335/12140651/888087b8-b461-11e5-86a6-7c8007aa91e7.jpg)
![20160106_103046](https://cloud.githubusercontent.com/assets/16563335/12140652/8880ea0a-b461-11e5-9a9a-7de95615bb73.jpg)
![20160106_103056](https://cloud.githubusercontent.com/assets/16563335/12140653/8882d20c-b461-11e5-8b0b-35103d063b5c.jpg)
![20160106_103103](https://cloud.githubusercontent.com/assets/16563335/12140654/8885296c-b461-11e5-9dc8-806d9154cb70.jpg)
![20160106_103107](https://cloud.githubusercontent.com/assets/16563335/12140655/8889c648-b461-11e5-811c-46abd8a5f67d.jpg)
![20160106_103120](https://cloud.githubusercontent.com/assets/16563335/12140656/888deed0-b461-11e5-9ff9-b4fcd2c47f75.jpg)
![20160106_103155](https://cloud.githubusercontent.com/assets/16563335/12140659/88a063bc-b461-11e5-81b7-1e138324329e.jpg)
![20160106_103203](https://cloud.githubusercontent.com/assets/16563335/12140657/8898141e-b461-11e5-87be-be6c76380dff.jpg)
![20160106_103210](https://cloud.githubusercontent.com/assets/16563335/12140658/889d9970-b461-11e5-9aed-16fe0c8eea86.jpg)

Here are some pics i hope they show all good but just showing i have slimdx. showing the dll are not blocked some errors and what is happening when i open it 

btw if it helps im running 64 bit windows 7 atm

**Comment by topher-au** [comment]
Ah yes, you need the 64 bit (x64) version of slimdx and the screen shot
shows 32 bit

**Comment by evans95** [comment]
:O i cant believe it haha how did i not notice problem solved :L haha thankyou

**Comment by Kyle757** [comment]
Hey, I can't get it to work. When I open WoWMapper it gives me an error pop-up.  I'm able to press continue and just go past it but when I press plug-ins it does the same error window over and over again. If I press details it shows me this. Not sure if it'll help. 

See the end of this message for details on invoking 
just-in-time (JIT) debugging instead of this dialog box.

************** Exception Text **************
System.IO.FileLoadException: Could not load file or assembly 'input_360, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515)
File name: 'input_360, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' ---> System.NotSupportedException: An attempt was made to load an assembly from a network location which would have caused the assembly to be sandboxed in previous versions of the .NET Framework. This release of the .NET Framework does not enable CAS policy by default, so this load may be dangerous. If this load is not intended to sandbox the assembly, please enable the loadFromRemoteSources switch. See http://go.microsoft.com/fwlink/?LinkId=155569 for more information.
   at System.Reflection.RuntimeAssembly._nLoad(AssemblyName fileName, String codeBase, Evidence assemblySecurity, RuntimeAssembly locationHint, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.RuntimeAssembly.InternalLoadAssemblyName(AssemblyName assemblyRef, Evidence assemblySecurity, RuntimeAssembly reqAssembly, StackCrawlMark& stackMark, IntPtr pPrivHostBinder, Boolean throwOnFileNotFound, Boolean forIntrospection, Boolean suppressSecurityChecks)
   at System.Reflection.Assembly.Load(AssemblyName assemblyRef)
   at WoWmapper.PluginSelectForm.RefreshInputPlugins()
   at WoWmapper.PluginSelectForm..ctor()
   at WoWmapper.MainForm.buttonSelectPlugin_Click(Object sender, EventArgs e)
   at System.Windows.Forms.Control.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnClick(EventArgs e)
   at System.Windows.Forms.Button.OnMouseUp(MouseEventArgs mevent)
   at System.Windows.Forms.Control.WmMouseUp(Message& m, MouseButtons button, Int32 clicks)
   at System.Windows.Forms.Control.WndProc(Message& m)
   at System.Windows.Forms.ButtonBase.WndProc(Message& m)
   at System.Windows.Forms.Button.WndProc(Message& m)
   at System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)




************** Loaded Assemblies **************
mscorlib
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.NET/Framework64/v4.0.30319/mscorlib.dll
----------------------------------------
WoWmapper
    Assembly Version: 0.4.3.0
    Win32 Version: 0.4.3.0
    CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapper.exe
----------------------------------------
System.Windows.Forms
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Windows.Forms/v4.0_4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll
----------------------------------------
System
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System/v4.0_4.0.0.0__b77a5c561934e089/System.dll
----------------------------------------
System.Drawing
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Drawing/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll
----------------------------------------
System.Configuration
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Configuration/v4.0_4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll
----------------------------------------
System.Core
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Core/v4.0_4.0.0.0__b77a5c561934e089/System.Core.dll
----------------------------------------
System.Xml
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.dll
----------------------------------------
Newtonsoft.Json
    Assembly Version: 7.0.0.0
    Win32 Version: 7.0.1.18622
    CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/Newtonsoft.Json.DLL
----------------------------------------
WoWmapper_Input
    Assembly Version: 1.0.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapper_Input.DLL
----------------------------------------
WoWmapperEI
    Assembly Version: 0.3.0.0
    Win32 Version: 1.0.0.0
    CodeBase: file:///C:/Program%20Files%20(x86)/World%20of%20Warcraft/Interface/AddOns/WoWmapperEI.DLL
----------------------------------------
System.Numerics
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Numerics/v4.0_4.0.0.0__b77a5c561934e089/System.Numerics.dll
----------------------------------------
System.Runtime.Serialization
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Runtime.Serialization/v4.0_4.0.0.0__b77a5c561934e089/System.Runtime.Serialization.dll
----------------------------------------
System.Xml.Linq
    Assembly Version: 4.0.0.0
    Win32 Version: 4.0.30319.33440 built by: FX45W81RTMREL
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_MSIL/System.Xml.Linq/v4.0_4.0.0.0__b77a5c561934e089/System.Xml.Linq.dll
----------------------------------------
System.Data
    Assembly Version: 4.0.0.0
    Win32 Version: 4.6.1055.0 built by: NETFXREL2
    CodeBase: file:///C:/Windows/Microsoft.Net/assembly/GAC_64/System.Data/v4.0_4.0.0.0__b77a5c561934e089/System.Data.dll
----------------------------------------

************** JIT Debugging **************
To enable just-in-time (JIT) debugging, the .config file for this
application or computer (machine.config) must have the
jitDebugging value set in the system.windows.forms section.
The application must also be compiled with debugging
enabled.

For example:

<configuration>
    <system.windows.forms jitDebugging="true" />
</configuration>

When JIT debugging is enabled, any unhandled exception
will be sent to the JIT debugger registered on the computer
rather than be handled by this dialog box.


![untitled](https://cloud.githubusercontent.com/assets/16599745/12182556/d8bd5148-b556-11e5-8ac6-898dc8ca8d93.png)


![untitled](https://cloud.githubusercontent.com/assets/16599745/12182480/779bd97a-b556-11e5-81e8-43a9569c4e16.png)

**Comment by seblindfors** [comment]
@Kyle757 Did you not read the original post in here? This thread is for discussing ideas. If everyone reported their issues in the same issue, it would end up a complete mess after a while. Please remove your post and put it in a separate issue, but check the closed issues first before you do so.
